src/components/UI/dropdown/dropdown.module.scss

```diff
@@ -0,0 +1,143 @@
+// scss variables
+$border-color: #000000;
+$black: #000000;
+$text-color: #ffffff;
+$hover-color: #49a82e;
+$background-color: #ffffff;
+
+.userGreet {
+  color: $text-color;
+  margin: 23px;
+}
+
+.userGreet a {
+  padding: 0px;
+}
+
+.userProfilePic {
+  width: 32px;
+  height: 32px;
+  border-radius: 50%;
+  display: inline;
+  vertical-align: middle;
+}
+.dropdownHeader {
+  margin: 10px;
+  cursor: pointer;
+}
+
+.userGreetMsg {
+  display: inline;
+  vertical-align: middle;
+  margin-right: 10px;
+  color: rgb(255, 255, 255);
```

Comment:
why are we using RGB here instead of hex code?

Reply:
I didn't  added  it was there only I just copied it when I was making the sperate compoonent for drop down please let me know what can I do  or how can I change it .

Reply:
Even if you are copying it, it is a part of the change in your PR. So, it would be a good idea to write its hex code/variable name here

Reply:
Yess,I totally understood what you said Maheima, I will make the required changes.